src/Sylius/Bundle/CoreBundle/Controller/CheckoutController.php

```diff
@@ -194,11 +194,12 @@ public function finalizeAction(Request $request, OrderInterface $order)
     }
 
     /**
+     * @param int $id
+     *
      * @return Response
      */
-    public function thankYouAction()
+    public function thankYouAction($id)
```

Comment:
@adamelso is doing something similar for us, worth checking Adam?

He was also considering RBAC for a role to manage orders as well to be able to check how the confirmation looked for any order.